.github/workflows/build.yml

```diff
@@ -24,6 +24,8 @@ jobs:
         recipe:
           # non-userns
           # general
+          - recipes/general/recipe-aurora-main.yml
```

Comment:
if we're including the surface images for dx, we should also include the non-dx surface images here